index.d.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference types="jest"/>
```

Comment:
I'm not sure. I took [jest-immutable-matchers/index.d.ts](https://github.com/unindented/custom-immutable-matchers/blob/master/packages/jest-immutable-matchers/index.d.ts)  as an example cause it was working without any configuration with my IDE.